AlfrescoApp/View Controllers/FileFolder View Controller/BaseCollectionViewFlowLayout.m

```diff
@@ -108,7 +109,7 @@ - (void)prepareLayout
     CGFloat height = 0;
     if (self.itemHeight == -1)
     {
-        height = (self.collectionViewWidth - ((self.numberOfColumns + 1) * self.minimumInteritemSpacing)) / self.numberOfColumns;
+        height = (self.collectionViewWidth - ((self.numberOfColumns + 1) * self.minimumInteritemSpacing)) / self.numberOfColumns  * 1.3f;
```

Comment:
What's the reason for the 1.3x multiplier? Could you either document it via a comment or add it as a constant please?